programmers/20210722/기능개발.js

```diff
@@ -0,0 +1,18 @@
+function solution(progresses, speeds) {
+    let count = 0;
+    const result = [];
+    
+    while (count < speeds.length) {
+        for (let i = 0; i < speeds.length; i += 1) progresses[i] += speeds[i];
```

Comment:
```suggestion
        progresses.map((v, index) => v + speeds[index]);
```
for문을 사용하지 않고 map으로도 구현이 가능합니다
이게 시간이 덜 걸리더라구요

Reply:
배워갑니다